frontend/src/js/amazon.jsx

```diff
@@ -5,70 +5,44 @@ import PaymentStatus from './PaymentStatus/index.jsx'
 import SearchResults from './SearchResults/index.jsx'
 import ProductPage from './ProductPage/ProductPage.jsx'
 import Basket from './BasketEntry/index.jsx'
+import { browserHistory, Router, Route, IndexRoute } from 'react-router'
 import Checkout from './Checkout/index.jsx'
```

Comment:
I think PaymentStatus needs to be imported here from ./PaymentStatus/index.jsx - it's used below in the routes but not defined up here.